app/src/main/java/co/icanteach/projectx/InterviewApplication.kt

```diff
@@ -19,11 +24,29 @@ class InterviewApplication : Application(), HasActivityInjector {
 
     override fun onCreate() {
         super.onCreate()
+        Stetho.initialize(
+            Stetho.newInitializerBuilder(this)
+                .enableDumpapp(SampleDumperPluginsProvider(this))
+                .enableWebKitInspector(Stetho.defaultInspectorModulesProvider(this))
+                .build()
+        )
 
         DaggerAppComponent
             .builder()
             .app(this)
             .create(this)
             .inject(this)
     }
+
+    private class SampleDumperPluginsProvider(private val mContext: Context) :
```

Comment:
SampleDumperPluginsProvider should not be implemented in the InterviewApplication class.